<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSPIFlash</class>
 <widget class="QWidget" name="FormSPIFlash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonJedecId">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Read JedecId</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>addr (hex)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditReadAddr">
       <property name="text">
        <string>0</string>
       </property>
       <property name="maxLength">
        <number>12</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditReadSize">
       <property name="text">
        <string>8</string>
       </property>
       <property name="maxLength">
        <number>12</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReadData">
       <property name="text">
        <string>Read Data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sector (4KB) erase index</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSectorEraseIndex">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSectorErase">
       <property name="text">
        <string>Erase</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>addr (hex)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditWriteAddr"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWriteData">
       <property name="text">
        <string>WriteData</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>page (256 B) index</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPageIndex">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWritePage">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>WritePage</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonReadAll">
     <property name="text">
      <string>Read All</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonTest">
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelFlashStatus">
     <property name="text">
      <string>flash status</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
